README.md

```diff
@@ -38,4 +38,4 @@ resource "ohdear_site" "test" {
 ## Requirements
 
 * [Terraform](https://www.terraform.io/downloads.html) >= 0.12.x
-* [Go](https://golang.org/doc/install) 1.17 (for development)
+* [Go](https://golang.org/doc/install) 1.19 (for development)
```

Comment:
We could also say something about see `go.mod` for current version or just leave it blank